src/Razor/Microsoft.AspNetCore.Razor.Language/src/CodeGeneration/CodeWriterExtensions.cs

```diff
@@ -330,11 +330,36 @@ public static CodeWriter WriteAutoPropertyDeclaration(this CodeWriter writer, IL
             writer.Write(" ");
             writer.Write(propertyName);
             writer.Write(" { get; set; }");
+            writer.WritePropertyValue(typeName, defaultValue);
             writer.WriteLine();
 
             return writer;
         }
 
+        private static void WritePropertyValue(this CodeWriter writer, string typeName, string propertyValue)
+        {
+            if (propertyValue != null)
+            {
+                writer.Write(" = ");
+                switch (typeName)
+                {
+                    case "System.String":
+                        writer.WriteStringLiteral(propertyValue);
+                        break;
+                    case "System.Char":
+                        writer.Write(string.Join(string.Empty, "'", propertyValue, "'"));
```

Comment:
```suggestion
                        writer.Write("'" + propertyValue + "'");
```

Reply:
But the perf, Pranav! TBH, I think this is fine. Yeah, it's not a hot codepath but I actually think the `Join` reads a little bit better than concat via `+`.